api/src/constants/errors.ts

```diff
@@ -6,6 +6,8 @@ export class ErrorConstants {
     public static readonly VALIDATION_BODY_EMAIL = "The request body is missing the email field.";
     public static readonly VALIDATION_BODY_STUDENTNUMBER = "The request body is missing the studentNumber field.";
     public static readonly VALIDATION_BODY_USERID = "The request body is missing the userId field.";
+    public static readonly VALIDATION_BODY_CHECKIN = "The request body is missing the check in field";
+    public static readonly VALIDATION_BODY_CHECKOUT = "The request body is missing the check out field";
```

Comment:
should say `checkIn` and `checkOut` to be consistent with the other errors. This makes it easier to debug